WebApplication/Services/ForgeOSS.cs

```diff
@@ -248,8 +269,8 @@ public async Task<Autodesk.Forge.Client.ApiResponse<dynamic>> GetObjectAsync(str
         /// Copy OSS object.
         /// </summary>
         public async Task CopyAsync(string bucketKey, string fromName, string toName)
-        {
-            await WithObjectsApiAsync(async api => await api.CopyToAsync(bucketKey, fromName, toName));
+        {

```

Comment:
It is a real change - here we are not using the retry policies so that when the configurator is checking if a given configuration already exists it gets answer quicker instead of retrying again and again

Reply:
I am maybe blind, but line on left side looks exactly the same as line on the rights side. There is just some whitespace extra (let line 250 vs right line 270)

Reply:
Sorry, I was thinking of another one.